app/services/procedure_export_service.rb

```diff
@@ -73,10 +73,12 @@ def options_for(table, format)
       { instances: table.last, sheet_name: table.first }
     end.merge(DEFAULT_STYLES)
 
-    # transliterate: convert to ASCII characteres
+    # transliterate: convert to ASCII characters
     # to ensure truncate respects 30 bytes
-    options[:sheet_name] = I18n.transliterate(options[:sheet_name], '')
-      .truncate(30, omission: '')
+    # /\*?[] are invalid Excel worksheet characters
+    options[:sheet_name] = I18n.transliterate(
+      ActiveStorage::Filename.new(options[:sheet_name].delete('[]*?')).sanitized, '', locale: :en
+    ).truncate(30, omission: '')
```

Comment:
```suggestion
    options[:sheet_name] = I18n.transliterate(options[:sheet_name], '', locale: :en)
      .delete('/\*?[]')
      .truncate(30, omission: '')
```

transliterate peut toujours générer des caractères interdits, supprimons les après. Je ne suis pas sur que `ActiveStorage::Filename` apporte qqchose.

Reply:
On utilise `ActiveStorage::Filename` pour les champs répétition (`Champs::RepetitionChamp#libelle_for_export`) et il n'y a pas d'erreurs :
```
  # We have to truncate the label here as spreadsheets have a (30 char) limit on length.
  def libelle_for_export
    str = "(#{stable_id}) #{libelle}"
    # /\*?[] are invalid Excel worksheet characters
    ActiveStorage::Filename.new(str.delete('[]*?')).sanitized
  end
```

mais en effet là c'est ceintures, bretelles, airbag et parachute